src/components/actions/processTrigger.ts

```diff
@@ -30,7 +31,10 @@ export async function processTrigger(payload: TriggerPayload): Promise<ActionRes
   const actionsSequential = resolvedActions.filter(({ sequence }) => !!sequence)
   const actionsAsync = resolvedActions.filter(({ sequence }) => !sequence)
 
-  for (const action of [...actionsAsync, ...actionsSequential]) {
+  // Get core actions for the current trigger
+  const currentCoreActions = coreActions?.[trigger] ?? []
+
+  for (const action of [...actionsAsync, ...currentCoreActions, ...actionsSequential]) {
```

Comment:
Core actions get loaded the inserted into the action queue along with the rest of the sequential actions. From there on, there's no difference between them -- they get handled exactly the same)